packages/components/organisms/f-checkout/src/components/Notes.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <div>
+        <accordion
```

Comment:
When it's a collection or a dine-in order, are we going to have "delivery notes"?

Reply:
I explained above but short answer: yes. The current call we make hard codes the value to "delivery" so that is the type of thing where we need to make it backwards compatible. But as mentioned, I am going to adapt this for those changes before merging 🙂

Reply:
Updated now for new API changes